classes/page_group.php

```diff
@@ -34,6 +34,25 @@ class page_group extends persistent {
     /** Name of the cache used in this class. */
     public const CACHE_NAME = 'page_group_metadata';
 
+    /**
+     * Returns the current month in YYYYMM format.
+     * @return int
+     */
+    public static function get_current_month(): int {
+        return (int) userdate(time() - 360, '%Y%m');
```

Comment:
Can we add a description and clarification for what the 360 is used for here?

Reply:
I frankly don't know what it is for myself. I got rid of it.